ironfish/src/primitives/noteEncrypted.ts

```diff
@@ -4,13 +4,16 @@
 
 import {
   ENCRYPTED_NOTE_LENGTH,
+  ENCRYPTED_NOTE_PLAINTEXT_LENGTH,
+  MAC_LENGTH,
+  NOTE_ENCRYPTION_KEY_LENGTH,
   NoteEncrypted as NativeNoteEncrypted,
 } from '@ironfish/rust-nodejs'
 import bufio from 'bufio'
 import { Serde } from '../serde'
 import { Note } from './note'
 
-const PROOF_LENGTH = 192
+export const PROOF_LENGTH = 192
```

Comment:
Proofs aren't a property of notes, so I don't think it makes sense to export this constant from `noteEncrypted` -- maybe the Rust code could export it from somewhere general?

Reply:
ok

Reply:
Create proof length in output